src/components/CalendarMonth.jsx

```diff
@@ -44,6 +45,7 @@ const defaultProps = {
   onDayMouseEnter() {},
   onDayMouseLeave() {},
   renderDay: null,
+  firstDayOfWeek: moment.localeData().firstDayOfWeek(),
```

Comment:
this will freeze the default at parse time (and the locale could be changed at any time) - i think this default needs to be calculated in the render path, and have it default to `-1` or `null` here.